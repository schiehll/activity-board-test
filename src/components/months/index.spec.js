import React from 'react'
import renderer from 'react-test-renderer'
import Months from 'components/months'

describe('Months', () => {
  it('should render as expected', () => {
    const tree = renderer.create(<Months />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
