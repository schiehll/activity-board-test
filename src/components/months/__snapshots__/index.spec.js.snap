// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Months should render as expected 1`] = `
<div
  className="sc-bwzfXH iBbYAc"
>
  <span>
    oct
  </span>
  <span>
    nov
  </span>
  <span>
    dec
  </span>
  <span>
    jan
  </span>
  <span>
    feb
  </span>
  <span>
    mar
  </span>
  <span>
    apr
  </span>
  <span>
    may
  </span>
  <span>
    jun
  </span>
  <span>
    jul
  </span>
  <span>
    ago
  </span>
  <span>
    sep
  </span>
</div>
`;
